### DISCUSSION TITLE
Require Github Authentication + Real Name on Marketplace for Reviews

### DISCUSSION THREAD

**Discussion by ntotten** [microsoft/vsmarketplace#144] [Feedback]
The review system on the Marketplace for extensions borders on useless for popular. The [Prettier extension](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode&ssr=false#review-details) get a fairly high volume of reviews that generally fall into two categories 1) "this suck/is broken/don't like" and 2) "This is great/love it/thanks". These reviews add nothing to the community and do nothing to support the work of open source developers.

I believe there are a few things that could be done fairly easily to encourage higher quality reviews. One is to require the users to authenticate with Github and to always display their real name. Currently, reviewers are able to post anonymously which - as in everything on the internet - results in horrible behavior.

Simply requiring users to Authenticate with Github and display their name and link to their Github profile would encourage a more civil discussion. Github profiles are usually associated with developers' professional lives so they are treated more carefully. It won't be perfect, but I think this will help.

**Comment by sandy081** [comment]
cc @prashantvc

**Comment by prashantvc** [comment]
@ntotten We are aware of this problem. We are working on improving the review system, people do need to login to leave a review, and we use real names from their MS account. We need to evaluate if supporting GitHub Authentication will have any effect. Possibly do away with star rating al together
